src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -180,6 +209,260 @@ In the example above, the files required are placed within the `/tmp` directory.
 
 Refer to the set of `conduct` commands described in [[Deploying bundles|DeployingBundlesOps]] for more details.
 
+
+## Creating and deploying custom HAProxy configuration for Sandbox
+
+Follow these steps to provide custom HAProxy configuration to ConductR HAProxy running in the Sandbox. To provide a custom configuration for a non-sandbox environment go to the chapter [Creating and deploying custom HAProxy configuration](#creating-and-deploying-custom-haproxy-configuration).
+
+> The steps are the same as in non-sandbox mode. However, the configuration values differ from the non-sandbox environment since HAProxy is running as a docker container on the Sandbox.
+
+We will go through the building bundle configuration for HAProxy that can be loaded into the Sandbox. This will allow you to modify the bundle configuration as you see fit.
```

Comment:
> This will allow you to modify the bundle configuration as you see fit.

I think this sentence can be improved, maybe: "This will allow you to customize the behavior of HAProxy when using the Sandbox". You could also choose to remove the line in my opinion.